projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -623,7 +623,13 @@ public static function load_scripts_as_required( $type, $script_dependencies = a
 			// If this is a customizer preview, render the script directly to the preview after autosave.
 			// phpcs:ignore WordPress.Security.NonceVerification.Recommended
 			if ( is_customize_preview() && ! empty( $_GET['customize_autosaved'] ) ) {
-				echo '<script id="jetpack-block-' . esc_attr( $type ) . '" src="' . esc_attr( $view_script ) . '?ver=' . esc_attr( $script_version ) . '"></script>';
+				// The Map block is dependent on wp-element, and it doesn't appear to to be possible to load
+				// this dynamically into the customizer iframe currently.
+				if ( 'map' === $type ) {
+					echo '<div>' . esc_html_e( 'No preview available. Publish and refresh to see this widget', 'jetpack' ) . '</div>';
```

Comment:
Tiniest of tiny nits: should we add another full-stop at the end of the sentence?

Reply:
I agree, should have one at the end.

Reply:
I think it should also say "No map preview available". To be clearer in situations where there are multiple blocks.

Reply:
Done.